src/pages/search/advanced_search/PublisherFilters.tsx

```diff
@@ -0,0 +1,77 @@
+import { Autocomplete, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { t } from 'i18next';
+import { useState } from 'react';
+import { useHistory } from 'react-router';
+import { searchForPublishers } from '../../../api/publicationChannelApi';
+import { ResultParam } from '../../../api/searchApi';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { Publisher } from '../../../types/registration.types';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+
+export const PublisherFilters = () => {
+  const history = useHistory();
+  const [publisherQuery, setPublisherQuery] = useState('');
+  const debouncedQuery = useDebounce(publisherQuery);
+  const searchParams = new URLSearchParams(history.location.search);
+  const publisherParam = searchParams.get(ResultParam.Publisher);
+
+  const handleChange = (selectedValue: Publisher | null) => {
+    if (selectedValue) {
+      const publisherIdentifier = selectedValue.id.split('publisher/')[1].split('/')[0];
+      searchParams.set(ResultParam.Publisher, publisherIdentifier);
+    } else {
```

Comment:
Denne ble kanksje litt vel hacky, og har muligens noen fallgruver. Men blir uhorvelig laange URL'er uten😬

Reply:
Hadde `selectedValue.id.split('/')[1]` funket?

Reply:
https://api.dev.nva.aws.unit.no/publication-channels-v2/publisher/3A274BC8-5A40-461B-8895-F075BDF303EC/2023, så må nok dobbeltsplitte.

Reply:
litt dømt til å bli bugs av dobbel `split` og hardkodet `publisher/` da 🤔 har dere diskutert muligheten for å legge til `identifier` på alle kanaler?
På de fleste endepunktene i dag mener jeg vi får både `id` (URI) og `identifier` (UUID e.l.). Om vi hadde lagt inn det her også, kunne vi brukt bare `selectedValue.identifier` rett frem her, så sikrer APIet at vi jobber med riktig ID (uten årstall) uten at klienten må lage det selv

Reply:
Ah, da er nok løsningen din bedre ja

Reply:
Okei, så lenge alle er klare over svakhetene er det greit for meg :)
Kan være lurt å lage oppgave(r) på det da, bare for å ha noe å  peke på når APIet en dag ikke har med årstall by default i `id`, eller bruker noe annet enn `publisher/`, bare så det er synlig at vi ikke syns det var en god løsning når det snakkes om yet another frontend-bug 😅